### PR TITLE
Delete hotkey added

### PR DESCRIPTION
**Pull request by fkurmannucsc**
Added a delete hotkey to remove the uppermost item in the screenshot area.

### CODE REVIEW

File: static/js/main.js [hunk 1]
```diff
@@ -398,7 +398,11 @@ function takeScreenshot(source, x, y, width, height, format = 'image/jpeg') {
     return canvas.toDataURL(format);
 }
 
-function deleteScreenshot(screenshot_id) {
+function deleteScreenshot(screenshot_id=null) {
```
Comment (eriq-augustine) [inline static/js/main.js]: Missing spaces around `=`.

File: static/js/main.js [hunk 2]
```diff
@@ -398,7 +398,11 @@ function takeScreenshot(source, x, y, width, height, format = 'image/jpeg') {
...
+    // If no id is given, delete most recent screenshot.
+    if (screenshot_id == null) {
+        screenshot_id = document.querySelector('.screenshot-area .screenshot').getAttribute('data-id');
+    }
```
Comment (eriq-augustine) [inline static/js/main.js:405]: Missing space after this if.
We generally don't want executing code directly after a conditional (without vertical whitespace).

**Review by eriq-augustine — CHANGES_REQUESTED** [review]
I don't really think this hotkey should exist.
Since we don't currently have a concept of the "selected" screenshot, I don't think we should have a universal hotkey for it.

**Comment by fkurmannucsc** [comment]
> I don't really think this hotkey should exist. Since we don't currently have a concept of the "selected" screenshot, I don't think we should have a universal hotkey for it.

I agree that having a notion of which screenshot is selected would make sense. Perhaps the selected screenshots should be highlighted somehow and then with the up/down arrows users can navigate which screenshot is selected. May be a larger item but I could start to implement this.

**Comment by eriq-augustine** [comment]
> I agree that having a notion of which screenshot is selected would make sense. Perhaps the selected screenshots should be highlighted somehow and then with the up/down arrows users can navigate which screenshot is selected. May be a larger item but I could start to implement this.

I think you may have missed my point.

This PR doesn't make sense without the selection functionality.
And I am not convinced that we need the selection functionality.
So I think this PR should be dropped.

**Pull request closed without merging**